### DISCUSSION TITLE
Why does Redash "Changes" table not record all changes?

### DISCUSSION THREAD

**Discussion by guban2** [getredash/redash#7132] [Q&A]
There is a table "Changes", that holds data about changes done in Redash. 
In my case i realised that not all the changes are recorded there. For most of querries only the last change is recorded (for some querries i found 2 rows recorded, no idea why it's different, anyway should be much more ). 

Is it a normal behavior? Is there a way (any settings) to record all changes in this table?

![2024-08-26_17-51-43](https://github.com/user-attachments/assets/ebb110b8-1167-4103-8a9c-d6f8ba68a386)

**Comment by gaecoli** [comment]
It is not recommended to share the online address. Both the dashboard and the query record changes.

**Reply by beccadriever** [reply]
Commenting here because I am also having the same problem. I can see that a query was updated a month ago, but the only row in the changes table is from 2 years ago. @gaecoli any advice?
